utils/utilities.js

```diff
@@ -1,8 +1,28 @@
-const ethers = require("ethers");
 const readline = require("readline");
+const ethers = require("ethers");
+
+const { bigNumberify } = ethers.utils;
+
+const ETH_TOKEN = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
+const RAY = bigNumberify("1000000000000000000000000000"); // 10**27
```

Comment:
where are these constants used? Seems like they are only used in defi_deployer.js where they are also locally defined.

Reply:
They are imported by `makerV2Manager_loan.js`, `makerV2Manager_invest.js` and `makerManager_loan.js`  with the exception of the `USD_PER_xxx` variables which are not  actually imported by any file (my bad). I think it would actually make more sense for these variables to be exported from `defi-deployer.js` since they are only used by maker test scripts. I will just leave `ETH_TOKEN` in `utilities.js` and make all test files that define `ETH_TOKEN` import it from `utilities.js` instead.